blueprints/ember-frost-sort/index.js

```diff
@@ -1,22 +1,33 @@
+const blueprintHelper = require('ember-frost-core/blueprint-helper')
+
 module.exports = {
-  description: '',
-  normalizeEntityName: function () {},
+  afterInstall: function (options) {
+    const addonsToAdd = [
+      {name: 'ember-computed-decorators', target: '~0.3.0'},
+      {name: 'ember-frost-core', target: '^1.14.3'},
+      {name: 'ember-elsewhere', target: '~0.4.1'},
+      {name: 'ember-hook', target: '^1.4.1'},
+      {name: 'ember-prop-types', target: '^3.10.2'}
```

Comment:
If core handles the installation then I believe you'll only need `core` - the hook here will only determine installation, not consumption.  Consumption would be a factor of the resolver and `ember-cli` registering the addon, which should happen automagically if the packages are present

Give it a shot, but I'm pretty sure I'm correct about this